CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ add_library(stationresponse SHARED
 add_library(elementresponse SHARED
   ElementResponse.cc)
 
+target_link_libraries(stationresponse casa_casa casa_measures casa_tables casa_ms)
```

Comment:
This is not the right way of linking with the casacore libraries. Plus, line 37 already has link statements:
`
target_link_libraries(stationresponse ${CASACORE_LIBRARIES})
target_link_libraries(elementresponse ${CASACORE_LIBRARIES})
`
So this is unnecessary...